### ISSUE TITLE
Unabe to see the Docker white whale Icon in system tray

### ISSUE THREAD

**Issue by eliassal** [docker/for-win#7658]
I have Docker Desktop for windows Edge, I run it but can't see the whale in the  system tray in spite of the fact that I can run docker commands (pull, run....) in a powershell terminal, here is a snspashot of my docker running but no whale icon displayed
![dockericon](https://user-images.githubusercontent.com/8447864/87457054-14400780-c608-11ea-8944-1f3eb0e5867e.jpg)

Here is the out put of 
```
docker --version
Docker version 19.03.12, build 48a66213fe
```

  - [x] I have uploaded Diagnostics
  - Diagnostics ID:  4D36D48D-A561-465F-BE67-71230B5093B1/20200714170914

### Expected behavior
Whale icon shuld appear in system tray in order to be able to see the differnet menu items especially switch between linux/wndws container
### Actual behavior
Whale icon does not display
### Information

  - Windows Version: win 10 64 Enterprise
  - Docker Desktop Version:   Docker version 19.03.12, build 48a66213fe
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: NO, it is on a physical win 10 machine

### Steps to reproduce the behavior

  1. .double click the DD icon on the desktop or run from menu even as administrator..
  2. ...I get docker interface displayed (settings, Troubleshoot, sign in.... (as you can notice in the snapshot above)
3. go to system tray, no whale icon (I also went to task bar and turned DD on)

Here is also the output of 
`docker info`

```
Client:
 Debug Mode: false

Server:
 Containers: 20
  Running: 1
  Paused: 0
  Stopped: 19
 Images: 105
 Server Version: 19.03.12
 Storage Driver: windowsfilter
  Windows:
 Logging Driver: json-file
 Plugins:
  Volume: local
  Network: ics internal l2bridge l2tunnel nat null overlay private transparent
  Log: awslogs etwlogs fluentd gcplogs gelf json-file local logentries splunk syslog
 Swarm: inactive
 Default Isolation: hyperv
 Kernel Version: 10.0 18363 (18362.1.amd64fre.19h1_release.190318-1202)
 Operating System: Windows 10 Enterprise Version 1909 (OS Build 18363.900)
 OSType: windows
 Architecture: x86_64
 CPUs: 8
 Total Memory: 15.94GiB
 Name: salam7
 ID: LS53:UEUO:KLEG:FRJ5:PRO3:LFUU:3AFH:IUW5:IOMT:BZGA:57HR:IJXR
 Docker Root Dir: H:\dockerStorage
 Debug Mode: true
  File Descriptors: -1
  Goroutines: 47
  System Time: 2020-07-14T19:01:41.8037496+02:00
  EventsListeners: 2
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
 Product License: Community Engine
```

**Comment by stephen-turner** [comment]
That's odd, I've never seen that before. Is it listed in the Windows settings "Set which icons appear on the taskbar"?

**Comment by stephen-turner** [comment]
And does toggling it to On in the settings make any difference?

**Comment by eliassal** [comment]
This is what I did, however, I have Docker desktop on this machine and icon was displayed correctly until recently. Also, As I said I did an update to the latest edge version

**Comment by eliassal** [comment]
Stephen, while waiting to find the root of the issue, is there a powershell command to switch between windows and linux containers?

**Comment by stephen-turner** [comment]
I don't have an answer to the missing icon. I am on the latest Edge and the whale shows up fine. And we haven't had any other reports of it.

On your other question, no, sorry, you have to switch using the UI.  Are you switching frequently?

**Comment by eliassal** [comment]
Yesm I need to switch to linux contaniers as i have a .net core project and needs DD in linux mode

**Comment by anishr06** [comment]
Faced the same issue because I ignored WSL 2 installation. Here are the steps I performed.

1. Uninstalled Docker
2. Restart the machine
3. Install the docker
4. Install the WSL 2 which installed using a separate MSI update package.
5. Restart

Found the docker whale icon in the system tray.

![image](https://user-images.githubusercontent.com/10188912/89223933-81521780-d58c-11ea-86f7-c69afa3252b9.png)
